tests/functional/triggering/16-fam-expansion.t

```diff
@@ -31,7 +31,7 @@ workflow_run_ok "${TEST_NAME}" \
     cylc play --debug --no-detach --set="SHOW_OUT='$SHOW_OUT'" "${WORKFLOW_NAME}"
 #-------------------------------------------------------------------------------
 contains_ok "$SHOW_OUT" <<'__SHOW_DUMP__'
-  + (((1 | 0) & (3 | 2) & (5 | 4)) & (0 | 2 | 4))
+  + (((3 | 2) & (5 | 4) & (1 | 0)) & (2 | 4 | 0))
```

Comment:
Ah wait, it seems this order is not guaranteed?

Reply:
On it.

Reply:
This should really be integration tested which would handing unstable sorting easier.

I've added a sort to stabilise family iteration order #5279 to sort these functional tests out for now.